site/src/components/Resources/ResourceAvatar.tsx

```diff
@@ -37,18 +35,12 @@ export type ResourceAvatarProps = { resource: WorkspaceResource }
 export const ResourceAvatar: FC<ResourceAvatarProps> = ({ resource }) => {
   const hasIcon = resource.icon && resource.icon !== ""
   const avatarSrc = hasIcon ? resource.icon : getIconPathResource(resource.type)
-  const styles = useStyles()
 
-  return <Avatar className={styles.resourceAvatar} src={avatarSrc} />
+  return (
+    <Avatar colorScheme="darken">
+      <AvatarIcon>
+        <img src={avatarSrc} alt="" />
```

Comment:
Just to check I follow: the difference between `<Avatar src={src} />` and `<Avatar><img src={src} /></Avatar>` is that the former shows the icon only and the former shows the icon wrapped in a circle, is that right?

Reply:
The difference is: when you pass the image as a child, it does not add the `MuiAvatar-img` class to it so it is just an image inside of a rounded `div` without any style. My idea for the `AvatarIcon` is to make the image behaves like a `MuiIcon`. 

Thinking better, I have no idea why I decided to use a child instead of using the `AvatarIcon` directly like:
```tsx
// Instead of
<AvatarIcon>
   <img src={avatarSrc} alt="" />
</AvatarIcon>

// it should be
<AvatarIcon src={avatarSrc} />
```

Reply:
Ahhh ok yeah that makes sense!